CONTRIBUTING.md

```diff
@@ -85,38 +111,127 @@ bolt build && \
 bolt version-packages && \
 bolt format && \
 git add . && \
-git commit -m "Prepare release" && \
+git commit -m "Version packages" && \
 git push --set-upstream origin temp-release-branch
 ```
 
 Once you have run this you will need to make a pull request to merge this back into master.
 
-#### Release Process
-
-Once the version changes are merged back in to master, to do a manual release:
+Finally, make sure you've got the latest of everything locally
 
 ```sh
 git checkout master && \
 git pull && \
-bolt && \
-bolt publish-changed && \
-git push --tags && \
 bolt
 ```
 
-The `bolt publish-changed` command finds packages where the version listed in the `package.json` is ahead of the version published on npm, and attempts to publish just those packages.
-
-Because of this, we should keep the following in mind:
-
-- Once the `publish-changed` command has been run, the PR from the `temp-release-branch` should be merged before any other PRs are merged into master, to ensure that no changesets are skipped from being included in a release.
-- There is no reason you should ever manually edit the version in the `package.json`
-
 ### A quick note on changelogs
 
 The release process will automatically generate and update a `CHANGELOG.md` file, however this does not need to be the only way this file is modified. The changelogs are deliberately static assets so past changelogs can be updated or expanded upon.
 
 In addition, content added above the last released version will automatically be appended to the next release. If your changes do not fit comfortably within the summary of a changelog, we encourage you to add a more detailed account directly into the `CHANGELOG.md`.
 
+### Backporting Fixes
+
+Occasionally a bug goes undetected for a few versions. When a fix is discovered,
+it needs to be applied to all affected versions. This is called _backporting_.
```

Comment:
```suggestion
it may need to be applied to all affected versions (depending on the
severity, security considerations, etc). This is called _backporting_.
```